resources/compass/charts/provisioner/templates/gardener-creds-secret.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.gardener.secretName }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Chart.Name }}
+    release: {{ .Release.Name }}
+type: Opaque
+data:
+  kubeconfig: {{ .Values.gardener.kubeconfig }}
```

Comment:
Can you please quote the base64 string with  {{ .Values.gardener.kubeconfig | quote }}